cpp/src_prims/random/make_arima.h

```diff
@@ -0,0 +1,253 @@
+/*
```

Comment:
nitpick: since this header file contains cuda kernel, as per the rules laid out in #1675 this should be renamed with a `.cuh` extension.